docs/docs/references/configuration/cli/trivy_kubernetes.md

```diff
@@ -64,11 +64,13 @@ trivy kubernetes [flags] { cluster | all | specific resources like kubectl. eg:
       --offline-scan                      do not issue API requests to identify dependencies
   -o, --output string                     output file name
       --parallel int                      number (between 1-20) of goroutines enabled for parallel scanning (default 5)
+      --password strings                  password. Comma-separated passwords allowed. TRIVY_PASSWORD should be used for security reasons.
```

Comment:
when is the password supposed to be used?

Reply:
if the image in cluster is stored in private registry